client/v2/README.md

```diff
@@ -209,6 +209,63 @@ For example, here is how the SDK does it for `cometbft` gRPC commands:
 https://github.com/cosmos/cosmos-sdk/blob/main/client/grpc/cmtservice/autocli.go#L52-L71
 ```
 
+## Off-Chain
```

Comment:
Nit, it is something different from autocli, we should put this below imho.
Ref: https://github.com/cosmos/cosmos-sdk/pull/18626#pullrequestreview-1836284353
The summary is part of autocli in that case.